### PR TITLE
Helper Function Refactors and improving readability

### PR DESCRIPTION
**Pull request by 0xdantrinh**
The small refactors we talked about in the app.ts Marcin PR that got merged in!

### CODE REVIEW

File: packages/commonwealth/client/scripts/state.ts [hunk 1]
```diff
@@ -238,6 +238,25 @@ const app: IApp = {
   cachedIdentityWidget: null,
 };
 
+function handleLoginSockets(data: any) {
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/state.ts:241]: We should properly type data here.

File: packages/commonwealth/client/scripts/helpers/chain.ts [hunk 2]
```diff
@@ -233,6 +218,25 @@ export const selectChain = async (
   return true;
 };
 
+export const initNewChain = async (newChain: any): Promise<boolean> => {
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/helpers/chain.ts:221]: and Type this helper as well

**Review by zakhap — CHANGES_REQUESTED** [review]

File: packages/commonwealth/client/scripts/state.ts (hunk 1 above)
Comment (0xdantrinh) [reply packages/commonwealth/client/scripts/state.ts:238]: Not sure how we would do that? Should we move all the status.ts types into a common-common types file?

Comment (0xdantrinh) [reply packages/commonwealth/client/scripts/state.ts:238]: i made the type more explicit like updateActiveUser, but this is also any

Comment (0xdantrinh) [reply packages/commonwealth/client/scripts/state.ts]: by adding data.result.user

**Comment by jnaviask** [comment]
What's the intent of this PR? Not clear to me what it is supposed to do -- "we talked about" isn't clear.

**Comment by 0xdantrinh** [comment]
Main purpose of this PR was to shorten the code in app.ts. When Marcin did his refactor of app.ts, these suggestions were given by me in my review. @CowMuon asked me to put these proposed changes in a separate PR.

**Pull request closed without merging**